src/backend/api/meals.js

```diff
@@ -2,13 +2,76 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+// http://localhost:5000/api/meals	GET,	Returns all meals
+router.get("/", async (req, res) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const meals = await knex("meal").select("id", "title");
+
+    meals.length === 0
+    ? res.status(404).send("Meals not found")
+    : res.status(200).send(meals);
+  } catch (error) {
+    res.status(500).json(error);
+  }
+});
+
+// /api/meals	POST, Adds a new meal to the database
+router.post("/", async (req, res) => {
+  try {
+    const meal = req.body;
+
+    const addMeal = await knex("meal").insert(meal);
```

Comment:
I would suggest two things here:
- first to add some validations before addding a meal
- second, if we are not using the variable 'addMeal', so it is better to not assign the response of knex to the 'addMeal' variable:
const addMeal = await knex("meal").insert(meal);  ==> await knex("meal").insert(meal);

and please consider it in the following endpoints :)